src/controls/tab-bar.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import { fieldOptionsType } from '../forms/helpers/field-prop-types'
+import { objectify, noop } from '../utils'
+
+/**
+ *
+ * @name TabBar
+ * @type Function
+ * @description A control component for navigating among multiple tabs
+ * @param {Boolean} [vertical] A boolean to determine whether the tabs should be aligned vertically or horizontally (optional, default `false`)
```

Comment:
Mention explicitly how it changes the class so if they're working with different CSS they know what changes to make